apps/web/src/server/lib/request.js

```diff
@@ -21,9 +21,9 @@ const instance = got.extend({
 	handlers: [
 		createTtlHandler(ttlCache),
 		(options, next) => {
-			if (!options.context.servedFromCache) {
-				console.log(`Sending ${kleur.bgBlue(options.method)} to ${kleur.blue(options.url)}`);
-			}
+			// if (!options.context.servedFromCache) {
```

Comment:
Debugging the issues we had last time was a pain without logging.

Please leave a TODO here to refactor once a more robust logging solution is added.

Reply:
Yeah, I left this commented by mistake, as it's nigh impossible to read test output with this on as the console gets filled with http messages... I think we need a similar kind of debug level thing like for auth